app/templates/events/view/tickets/discount-codes.hbs

```diff
@@ -0,0 +1,93 @@
+<div class="ui stackable grid">
+  <div class="row">
+    <div class="eight wide column">
+      <h2 class="ui header">
+        {{t 'Discount Codes'}}
+      </h2>
+    </div>
+    <div class="eight wide {{unless device.isMobile 'right aligned'}} column">
+      <button class="ui teal button">
+        <i class="plus icon"></i>
+        {{t 'Create a discount'}}
+      </button>
+    </div>
+  </div>
+  <div class="row">
+    <div class="ui hidden divider"></div>
+    <div class="ui {{if device.isMobile 'vertical'}} basic buttons">
+      <div class="ui button active">
+        {{t 'All'}}
+      </div>
+      <div class="ui button">
+        {{t 'Active'}}
+      </div>
+      <div class="ui button">
+        {{t 'Inactive'}}
+      </div>
+    </div>
+    <table class="ui very basic table">
+      <thead>
+        <tr>
+          <th>
+            #
+          </th>
+          <th>
+            {{t 'Discount code'}}
+          </th>
+          <th>
+            {{t 'Discount code URL'}}
+          </th>
+          <th>
+            {{t 'Discount Per Ticket'}}
+          </th>
+          <th>
+            {{t 'Validity'}}
+          </th>
+          <th>
+            {{t 'Status'}}
+          </th>
+          <th>
+            {{t 'Action'}}
+          </th>
+        </tr>
+      </thead>
+      <tbody>
+        {{#each model as |discount|}}
+          <tr>
+            <td>
+              {{discount.id}}
```

Comment:
This should be autogenerated. Won't be part of the API response. Use the loop's index to generate this.